src/components/summary/_summary.scss

```diff
@@ -190,6 +186,8 @@ $hub-row-spacing: 1.3rem;
 
         &__button {
             align-self: flex-start;
+            word-break: normal;
```

Comment:
This line isn't actually doing anything so we can remove it

Reply:
removed